audit/projects/kubernetes-public/services/dns/info.json

```diff
@@ -4,7 +4,6 @@
   "number": "127754664067",
   "quota": {
     "dnsKeysPerManagedZone": 4,
-    "gkeClustersPerPolicy": 100,
```

Comment:
I don't think this is anything we did, guessing a quota type was removed from google cloud